default_handler.go

```diff
@@ -38,12 +38,29 @@ type defaultHandler struct {
 	defaultIntf map[string]*exportedIntf
 }
 
-func (h *defaultHandler) PathExists(path ObjectPath) bool {
-	_, ok := h.objects[path]
-	return ok
+func (h *defaultHandler) GetExportedObject(path ObjectPath) (*exportedObj, bool) {
+	h.RLock()
+	defer h.RUnlock()
+	obj, ok := h.objects[path]
+	return obj, ok
+}
+
+// GetAnExportedObject returns an exportedObj for an specific ObjectPath
+// A new exportedObj is creaated if none existed for ObjectPath
+func (h *defaultHandler) GetAnExportedObject(path ObjectPath) *exportedObj {
```

Comment:
nit: would name this e.g. `GetOrAddExportedObject` to make it more clear what the difference to `GetExportedObject` is

Reply:
nit accepted.